R/RTemplate.R

```diff
@@ -173,9 +178,10 @@ RTscript = function() {
     )
   }
 
+  includedirs = if (opt$includedir != "") strsplit(opt$includedir,",")[[1]] else NULL
+
   if (opt$includedir != "") {
-    opt$includedir = strsplit(opt$includedir,",")[[1]]
-    addcode = c(addcode, paste("add.include.dir(\"",opt$includedir,"\")",sep=""))
+    addcode = c(addcode, paste("add.include.dir(\"",includedirs,"\")",sep=""))
```

Comment:
I think that now that `RTfile` has the `includedirs` argument, we can move this to `RTfile`.

Reply:
Actually, I think it is better to keep it here, since RTfile (and `RTconvert.R' in general) mainly does the code parsing and most of the code involved modifying / appending stuff to R file in inside `Rtemplate.R`